shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -38,13 +73,13 @@
 @ThreadSafe
 @PublicApi
 public final class LoadCommand extends AbstractFileSystemCommand {
-  private static final Option LOCAL_OPTION =
-      Option.builder()
-          .longOpt("local")
-          .required(false)
-          .hasArg(false)
-          .desc("load the file to local worker.")
-          .build();
+  // todo(jianjian) add option for timeout and parallelism(compatibility?)
+  private static final int DEFAULT_PARALLELISM = 4;
+
+  private static final Option LOCAL_OPTION = Option.builder().longOpt("local").required(false)
```

Comment:
nit: old formatting was neater. It helps to see when options are laid out over lines. 
BTW, I know our code-style file is not considerate to this, so I end up formatting line by line :(